### ISSUE TITLE
리딩: Hands-on ML 2: chap 11 - Training DNN's:  The vanishing / exploding gradient problems

### ISSUE THREAD

**Issue by eubinecto** [eubinecto/k4ji_ai#16]

**Comment by eubinecto** [comment]
gradient descent공식:
- 

problem|원인|결과|학습에 주는 영향
--|--|--|--
vanishing gradient| the loss gradient is close to zero| virtually no change in the weights after gradient descent because the "step" is too small|The weight does not converge at all. learning rate를 너무 작게 둔 것과 같은 효과.
exploding gradient| the loss gradient is too large|  too much change in the weights after gradient descent because "step" is too huge, inconsistent| The weight diverges. learning rate를 너무 크게 둔 것과 같은 효과.

**Comment by eubinecto** [comment]
## Observation of  exploding & vanishing gradients
from the paper, [*Understanding Difficulty of Training Deep Feedforward Neural Networks*](http://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf)
![image](https://user-images.githubusercontent.com/56193069/90420487-fbd57800-e0f2-11ea-82ee-66448bc662b6.png)
